vg/Makefile

```diff
@@ -4,7 +4,7 @@ runtime_fullpath = $(realpath runtime)
 build_tool = runtime-container.DONE
 git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../vg | cut -f1 -d " ")
 name = quay.io/ucsc_cgl/vg
-tag = 1.4.0--${git_commit}
+tag = 1.4.0--cb68327d76dd718dd155a6462866e65b093d4644
```

Comment:
If you do that, you also need to change the DOckerfile to checkout commit cb68327, of course.